railties/test/commands/secrets_test.rb

```diff
@@ -17,8 +17,21 @@ def teardown
     assert_match "No $EDITOR to open decrypted secrets in", run_edit_command(editor: "")
   end
 
+  test "edit secrets" do
+    run_setup_command
+    run_edit_command(editor: "ex -c :1d -c %s/#//g -c wq")
+    output = Dir.chdir(app_path) do
+      `bin/rails runner "puts Rails.application.secrets.external_api_key" -e production`
+    end
+    assert_equal "1466aac22e6a869134be3d09b9e89232fc2c2289…\n", output
```

Comment:
If we really want to do a true edit (I'm not sure how useful that is), we could try to stash a consistent key in `RAILS_MASTER_KEY`, then run `run_setup_command` and simulate an edit through something like `run_secrets_command(editor: "echo 'cheese' >")`.

I haven't been able to get the `echo` above to work locally, but I think we might.